server/routes/induction/exemption/exemption/index.ts

```diff
@@ -1,56 +1,61 @@
 import { Router } from 'express'
 import { Services } from '../../../../services'
 import asyncMiddleware from '../../../../middleware/asyncMiddleware'
-import createEmptyInductionExemptionDtoIfNotInPrisonerContext from '../../../routerRequestHandlers/createEmptyInductionExemptionDtoIfNotInPrisonerContext'
+import createEmptyInductionExemptionDtoIfNotInJourneyData from '../../../routerRequestHandlers/createEmptyInductionExemptionDtoIfNotInJourneyData'
 import ExemptionReasonController from './exemptionReasonController'
 import ConfirmExemptionController from './confirmExemptionController'
-import checkInductionExemptionDtoExistsInPrisonerContext from '../../../routerRequestHandlers/checkInductionExemptionDtoExistsInPrisonerContext'
+import checkInductionExemptionDtoExistsInJourneyData from '../../../routerRequestHandlers/checkInductionExemptionDtoExistsInJourneyData'
 import retrieveInductionSchedule from '../../../routerRequestHandlers/retrieveInductionSchedule'
 import ExemptionRecordedController from './exemptionRecordedController'
 import checkInductionIsScheduled from '../../../routerRequestHandlers/checkInductionIsScheduled'
 import { checkUserHasPermissionTo } from '../../../../middleware/roleBasedAccessControl'
 import ApplicationAction from '../../../../enums/applicationAction'
+import setupJourneyData from '../../../routerRequestHandlers/setupJourneyData'
 
 /**
  * Route definitions for exempting a prisoner's Induction
  */
 export default (services: Services) => {
-  const { auditService, inductionService } = services
+  const { auditService, inductionService, journeyDataService } = services
 
   const exemptionReasonController = new ExemptionReasonController()
   const confirmExemptionController = new ConfirmExemptionController(inductionService, auditService)
   const exemptionRecordedController = new ExemptionRecordedController()
 
   const router = Router({ mergeParams: true })
 
-  router.use([checkUserHasPermissionTo(ApplicationAction.EXEMPT_INDUCTION)])
+  router.use([
+    //
+    checkUserHasPermissionTo(ApplicationAction.EXEMPT_INDUCTION),
+    setupJourneyData(journeyDataService),
+  ])
 
   router.get('/', [
     checkInductionIsScheduled(inductionService), // Induction Schedule must be SCHEDULED in order to exempt it
-    createEmptyInductionExemptionDtoIfNotInPrisonerContext,
+    createEmptyInductionExemptionDtoIfNotInJourneyData,
```

Comment:
The "create...." and "check...." middlewares have been renamed, hence the changes here (previously they referenced the prisonerContext, now they reference the journeyData)